packages/config-utils/proxy.js

```diff
@@ -38,6 +38,7 @@ module.exports = ({
   onBeforeSetupMiddleware = () => {},
   bounceProd = false,
   useAgent = true,
+  useDevBuild = true,
```

Comment:
Shouldn't this be default as `false`? So when running on stage.foo you'd get dev React by default (no way to opt-out of this) and by default prod.foo will give you the prod React unless you specify `useDevBuild: true`?

Reply:
I would argue that for development you want a development bundle regardless of the environment. Plus folks that use prod env for development (there are a few of them) never actually test pre-prod chrome and are always surprised when something changes.

Reply:
The problem I have with this approach is that it could use outdated version of prod environent.

Reply:
Would it maybe be possible to show an indicator in the UI (only visible in dev) which version, maybe even commit of chrome is currently used?

Reply:
@bastilian  That is something we can do yeah. But that would have to be done in chrome. 

@karelhala it will always use the latest stable/beta version.